README.md

```diff
@@ -65,6 +79,87 @@ To generate coverage results, set the `mapCoverage` property in the `jest` confi
 
 > Please note that the `outDir` property in the `jest` configuration section is removed in coverage mode, due to [#201](https://github.com/kulshekhar/ts-jest/issues/201).
 
+## Sensible Defaults
+`ts-jest` tries to ship with sensible defaults, to get you on your feet as quickly as possible.
+
+### Sourcemap support
+Sourcemaps should work out of the box. That means your stack traces should have the correct line numbers,
+and you should be able to step through the TypeScript code using a debugger.
+
+### Automatically finds tsconfig.json
+`ts-jest` automatically located your `tsconfig` file.
+If you want to compile typescript with a special configuration, you [also can](#tsconfig)
+
+### Supports synthetic modules 
+If you're on a codebase where you're using synthetic default imports, e.g. 
+```javascript 1.6
+//Regular imports
+import * as React from 'react';
+
+//Synthetic default imports:
+import React from 'react';
+```
+`ts-jest` tries to support that. If `allowSyntheticDefaultImports` is set to true in your `tsconfig` file, it uses babel
+to automatically create the synthetic default exports for you - nothing else needed. 
+You can opt-out of this behaviour with the [skipBabel flag](#skipping-babel)
+
+### Supports automatic of jest.mock() calls
+[Just like Jest](https://facebook.github.io/jest/docs/manual-mocks.html#using-with-es-module-imports) `ts-jest`
+automatically hoists your `jest.mock()` calls to the top of your file.
+You can opt-out of this behaviour with the [skipBabel flag](#skipping-babel)
```

Comment:
It might be helpful to add a note about why users might want to skip this - skipping this could slightly speed up things because only typescript is used to transpile the code.